src/.vscode/launch.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Do not include your local launch setting in PR. Delete this one.

Reply:
Will do.